UnitTests/DbContextExtensions.cs

```diff
@@ -11,21 +11,29 @@ public static void Seed(this EducationLevelContext dbContext)
             dbContext.EducationLevels.AddRange(new EducationLevel
             {
                 Id = 1,
-                Value = "Less than High School"
+                Value = "University Degree"
             }, new EducationLevel
             {
                 Id = 2,
-                Value = "High School"
+                Value = "College Diploma or Apprenticeship, 2 or more years"
             }, new EducationLevel
             {
                 Id = 3,
-                Value = "Diploma/Certificate"
+                Value = "College Diploma or Apprenticeship, less than 2 years"
             }, new EducationLevel
             {
                 Id = 4,
-                Value = "Degree"
+                Value = "High School Diploma"
+            }, new EducationLevel
+            {
+                Id = 5,
+                Value = "No Formal Education"
+            },new EducationLevel
```

Comment:
If these ids will be sent to the backend for querying TEER levels, they should match the backend TEERs (e.g. Management == TEER 0)